apps/mobile/src/contexts/GalleryEditor/GalleryEditorContext.tsx

```diff
@@ -353,15 +377,48 @@ const GalleryEditorProvider = ({ children, queryRef }: Props) => {
           id: tokenId,
           kind: 'token',
           tokenRef,
+          communityName: tokenRef.definition.community?.name ?? '',
+          communityDescription: tokenRef.definition.community?.description ?? '',
+          communityId: tokenRef.definition.community?.dbid ?? '',
         });
       });
 
-      updateRow(activeRowId, (previousRow) => {
+      // Check if all the tokensToAdd have the same community
+      const communityIds = new Set(tokensToAdd.map((token) => token.communityId));
+
+      // if there are only tokens from the same community, then the section name should be the community name
+      if (communityIds.size === 1 && tokensToAdd[0]) {
+        sectionName = tokensToAdd[0].communityName;
+        sectionCollectorsNote = tokensToAdd[0].communityDescription;
+      }
```

Comment:
If the user only select token from the same community, we wanna populate the community name and description into editor